src/hooks/useMessages.tsx

```diff
@@ -10,7 +10,7 @@ export function useMessages(conversation: Conversation): Message[] | undefined {
   useEffect(() => {
     if (!client) return;
     loadMessages(conversation, client);
-  });
+  }, [client, conversation]);
```

Comment:
this was the issue with the reactions popping in and out. this effect was running way too often and double processing messages.

Reply:
Ahhhh, thanks for catching that!